optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/PhaseConfigForm.java

```diff
@@ -80,15 +78,15 @@ public void setModel( List<PhaseConfigModel> model ) {
         }
     }
 
-    @Override
-    public HTMLElement getElement() {
-        return view.getElement();
-    }
-
     @PreDestroy
     public void destroy() {
         for ( Object phaseForm : phaseFormList ) {
             syncBeanManager.destroyBean( phaseForm );
         }
     }
+
+    @Override
```

Comment:
@mbarkley When using IsElement implementation, drop down menu events in PhaseConfigFormViewImpl were not caught after the refactoring. This provided me a quick workaround. Do you think I might have missed something?